### PR TITLE
Optionally hide completed requests

### LINKED ISSUE #77: optionally hide completed requests

### PR DESCRIPTION
**Pull request by lowellbander**
resolves #77

### CODE REVIEW

**Comment by lowellbander** [comment]
@bluquar can you advise me how to:

1. not have to run `npm run dev:client && npm start` every time I make changes? i figure something with expo, but with expo I had all the nonsense about not having environment variables set
2. have the query respect the state change? do I need to be using an effect hook or something like that?
3. regenerate the client-side graphql code to include `filter` as a param to the `allAidRequests` query?

File: src/client/aid-app/aid_requests/ListOfAidRequests.tsx [hunk 1]
```diff
@@ -33,6 +38,13 @@ const LIST_OF_AID_REQUESTS_QUERY = gql`
 `;
 
 export default function ListOfRequests(): JSX.Element {
+  const [isHidingCompletedRequests, setIsHidingCompletedRequests] =
+    React.useState(true);
+
+  const filter = {
+    ...(isHidingCompletedRequests ? { completed: false } : {}),
+  };
```
Comment (bluquar) [inline src/client/aid-app/aid_requests/ListOfAidRequests.tsx:46]:
```suggestion
  const filter isHidingCompletedRequests
    ? {completed: false}
    : {};
```

**Comment by bluquar** [comment]
1. `expo start --web` allows you to make client-side changes and have them automatically reload. Unfortunately I haven't been able to figure out how to get GraphQL queries to work when using this method. The problem is that when you do this, the GraphQL server runs on one port (e.g., 3000) and the expo web build server runs on a different port (e.g., 19006.) Something about CORS causes either the server or client to reject the requests from 19006 to 3000. This is the issue you and I were debugging in person a week or two ago, and I still haven't figured out how to get around it. In order to make server-side changes and have them automatically re-build, we could use nodemon. I haven't set that up yet.
2. I think you need to change `src/client/aid-app/graphql/client.ts` to have `keyArgs: ['filter']` on line 18.
3. `npm run build:graphql`. Make sure you have the GraphQL server running on port 3000 (or change the command in package.json to point to wherever your GraphQL server is running.)

**Comment by lowellbander** [comment]
#89 does this

**Pull request closed without merging**